src/synapse/dataProvider.js

```diff
@@ -102,6 +102,16 @@ const resourceMap = {
     }),
     data: "members",
   },
+  pushers: {
+    map: p => ({
+      ...p,
```

Comment:
needs an `id` field. e.g. `map: p => ({
      ...p,
      id: p.app_id,
    }),`

but I think app_id is not a good id here

Reply:
I have it changed. `pushkey` is the unique identifier.
=> https://matrix.org/docs/spec/client_server/latest#post-matrix-client-r0-pushers-set